docs/current/guides/snippets/azure-pipelines-container-instances/main.py

```diff
@@ -0,0 +1,97 @@
+import sys
+import os
+
+import anyio
+import dagger
+from azure.identity import DefaultAzureCredential
+from azure.mgmt.containerinstance import ContainerInstanceManagementClient
+
+# configure container group, name and location
+container_name = "my-app";
+container_group_name = "my-app";
+container_group_location = "eastus";
+resource_group_name = "my-group";
+
+async def main():
+
+    # check for required variables in host environment
+    for var in ["DOCKERHUB_USERNAME", "DOCKERHUB_PASSWORD", "AZURE_SUBSCRIPTION_ID", "AZURE_TENANT_ID", "AZURE_CLIENT_ID", "AZURE_CLIENT_SECRET"]:
+        if var not in os.environ:
+            raise EnvironmentError('"%s" environment variable must be set' % var)
+
+    # initialize Dagger client
+    async with dagger.Connection(dagger.Config(log_output=sys.stderr)) as dagger_client:
+
+        # set registry password as Dagger secret
+        secret = dagger_client.set_secret("password", os.environ.get("DOCKERHUB_PASSWORD"));
```

Comment:
The default for `os.environ.get` when a key isn't found is to return `None`. Even though the env var is ensured to exist because of the early checks above, the IDE may complain (e.g., Pyright, which is a Python type checker):

<img width="798" alt="Screenshot 2023-05-30 at 17 28 13" src="https://github.com/dagger/dagger/assets/174525/2a92ca51-fe38-4249-9e11-2a36566803f2">

As it's already validated, you can just access like this instead: `os.environ["DOCKERHUB_PASSWORD"]`.